### PR TITLE
#20 Environment Variables

### PR DESCRIPTION
**Pull request by juliavisserinbibo**
## Issue
#20 

## Description
This issue has a list of the environment variables that can be added to Shift.
- SHIFT_PATH_WORKFLOWS
- SHIFT_PATH_PREFS
- SHIFT_PROCESS_<software>
- SHIFT_CATALOG_PATH
- SHIFT_PLUGIN_PATH
- SHIFT_MODE_DEV
- SHIFT_LOG_LEVEL
- SHIFT_LOG_MAXSIZE
- SHIFT_LOG_MAXFILES
- SHIFT_LOG_DIRECTORY

### CODE REVIEW

**Review by leegeertseninbibo — APPROVED** [review]

**Comment by leegeertseninbibo** [comment]
Please remove `SHIFT_ACTIVATION_SCRIPT` from this list. This should not be documented

File: docs/reference/environment.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Environment Variables
+
+This page lists the environment variables available for Shift to serve as a guide to set the a custom environment for Shift.
+
+| Environment Variable | Description |
+| :--- | :---------- | 
+| **SHIFT_PATH_WORKFLOWS** | Directory paths set in this variable will be considered as workflow project locations. This means that workflows existing in these directories will use relative paths when being referenced in [Workflow](nodes/workflow#workflow-node) and [WorkflowProcess](nodes/workflow#workflowprocess-node) nodes.|
+| **SHIFT_ACTIVATION_SCRIPT** | This variable should to an activation script to be run before a workflow is executed in  [Batch](../getting_started/basics/batch.md) mode.
```
Comment (albertochimentiinbibo) [inline docs/reference/environment.md:8]: @juliavisserinbibo as I updated in this comment https://github.com/Inbibo/shift/pull/1115#discussion_r1738545212 This environment variable should not be documented.

**Review by albertochimentiinbibo — CHANGES_REQUESTED** [review]

File: docs/reference/environment.md (hunk 1 above)
Comment (juliavisserinbibo) [reply docs/reference/environment.md]: Done

File: docs/reference/environment.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Environment Variables
+
+This page lists the environment variables available for Shift to serve as a guide to set the a custom environment for Shift.
+
+| Environment Variable | Description |
+| :--- | :---------- | 
+| **SHIFT_PATH_WORKFLOWS** | Directory paths set in this variable will be considered as a workflow project locations. This means that workflows existing in these directories will use relative paths when being referenced in [Workflow](nodes/workflow#workflow-node) and [WorkflowProcess](nodes/workflow#workflowprocess-node) nodes.|
+| **SHIFT_PATH_PREFS** | If a directory path is specified in this variable, the folder where the user preference file will be located in that directory.|
+| **SHIFT_PROCESS_<software>** | This variable is used to set the path locations of the Python interpreters used by the [WorkflowProcess] node. The name Shift will give to the interpreter will depend on the last word used in the environment variable name.|
```
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/reference/environment.md]: Done

**Review by hectorjimenezinbibo — APPROVED** [review]

**Review by albertochimentiinbibo — APPROVED** [review]

**Pull request merged**